lgc/patch/NggPrimShader.cpp

```diff
@@ -2179,7 +2179,8 @@ void NggPrimShader::buildPrimShaderWithGs(Function *entryPoint) {
     if (m_pipelineState->enableSwXfb()) {
       const auto &streamXfbBuffers = m_pipelineState->getStreamXfbBuffers();
       for (unsigned i = 0; i < MaxGsStreams; ++i) {
-        bool streamActive = streamXfbBuffers[i] != 0;
+        // Treat the vertex stream as active if it is associated with XFB buffers or is the rasterization stream.
```

Comment:
the comment doesn't match with the code change. could you remove the comment or refine it? thanks!

Reply:
Oh, my carelessness.